scrapscript.py

```diff
@@ -3330,7 +3330,7 @@ def serve_command(args: argparse.Namespace) -> None:
         server = socketserver.TCPServer
     server.allow_reuse_address = True
     with server(("", args.port), ScrapReplServer) as httpd:
-        print("serving at port", args.port)
+        print(f"serving at http://localhost:{args.port}")
```

Comment:
It may not always be localhost (for example, on fly.io)

Reply:
(also, unrelated to rest of change imo)

Reply:
It's a nice feature of some webservers I use (FastAPI, vite, nextjs) that it prints out the localhost address in the terminal. Then you can just command click on that address to open in browser. Easier than typing it all out.

But I will remove it since it's unrelated to styling.

Reply:
Updated in my branch because you can use `http.server_address` and then you get the nice debug print

Reply:
```patch
     with server(("", args.port), ScrapReplServer) as httpd:
-        print("serving at port", args.port)
+        host, port = httpd.server_address
+        print(f"serving at http://{host!s}:{port}")
         httpd.serve_forever()
```

Reply:
lmk what you think

Reply:
awesomeee